test/unit/JobRolesServiceTests.ts

```diff
@@ -0,0 +1,41 @@
+import axios from "axios";
+import MockAdapter from "axios-mock-adapter";
+import { expect } from 'chai';
+import { getJobs, URL } from '../../src/services/JobRoleService';
+import { JobRole } from "../../src/models/JobRole";
+import { describe, it } from "node:test";
+
+const jobRole: JobRole = {
+  jobRoleId: 1,
+  roleName: "Graduate Software Engineer",
+  location: "Derry",
+  capabilityId: 1,
+  bandId: 2,
+  Date: new Date(1693078000000),
+}
+
+const mock = new MockAdapter(axios);
+
+describe('JobRoleService', function () {
+    describe('getJobs', function () {
+      it('should return jobs from response', async () => {
+        const data = [jobRole];
+
+        mock.onGet(URL).reply(200, data);
+
+        const results = await getJobs();
+        expect(new Date(results[0].Date)).to.deep.equal(jobRole.Date);
```

Comment:
we should check the other fields as well

Reply:
For the AC on the View Job Role - https://tasks.office.com/kainos.com/en-GB/Home/Planner/#/plantaskboard?groupId=080e2153-5cc9-49af-8b98-3dc65ffde1fa&planId=IQhhunCtwUSxDbRmlZCQCJYAA0or, It's only looking for the id, role name, location, capabilityId , bandId and Date. It's extended in the view job role information ticket.

Reply:
Updated